pages/common/iotop.md

```diff
@@ -1,7 +1,7 @@
 # iotop
 
 > Display a table of current I/O usage by processes or threads.
-> More information: <https://linux.die.net/man/1/iotop>.
+> More information: <https://manpages.debian.org/iotop/iotop.8.html>.
```

Comment:
I prefer this:
```suggestion
> More information: <https://man7.org/linux/man-pages/man8/iotop.8.html>.
```

Reply:
Good to know&mdash;so there are more man7.org pages than I thought. I will be more careful with the rest of the 30 die.net pages 👍🏻